tests/test_other.py

```diff
@@ -9468,6 +9477,15 @@ def test_asan_null_deref(self):
       'AddressSanitizer: null-pointer-dereference on address',
     ])
 
+  @no_fastcomp('asan is not supported on fastcomp')
+  def test_asan_no_stack_trace(self):
+    self.do_smart_test(path_from_root('tests', 'other', 'test_lsan_leaks.c'),
+                       emcc_args=['-fsanitize=address', '-s', 'ALLOW_MEMORY_GROWTH=1', '-DDISABLE_CONTEXT', '-s', 'EXIT_RUNTIME'],
+                       assert_returncode=None, literals=[
+      'Direct leak of 3427 byte(s) in 3 object(s) allocated from:',
+      'SUMMARY: AddressSanitizer: 3427 byte(s) leaked in 3 allocation(s).',
```

Comment:
Is this the entire error message? Doesn't seem very useful :( Does it work for other ASan errors besides memory leaks?

Reply:
If you disable stack traces, you lose stack traces, unfortunately. But the speed increase might be worth it.

For other errors, it looks something like this:

```
=================================================================
==42==ERROR: AddressSanitizer: heap-use-after-free on address 0x03200f54 at pc 0x00000c41 bp 0x02965dc0 sp 0x02965dcc
READ of size 4 at 0x03200f54 thread T0
    #0 0xc41 in main /some/where/useafterfree.c:6:10
    #1 0x8000198a in Object.Module._main /some/where/a.out.js:6538:32

0x03200f54 is located 4 bytes inside of 40-byte region [0x03200f50,0x03200f78)
freed by thread T0 here:

previously allocated by thread T0 here:

SUMMARY: AddressSanitizer: heap-use-after-free /some/where/useafterfree.c:6:10 in main
```

Reply:
Ok, so if I understand correctly it's at least giving you the location of the error, so this is still a helpful error message.

Reply:
Yes, it gives you the location of memory errors, but not memory leaks.